scripts/render_configs.sh

```diff
@@ -130,25 +142,35 @@ else
 fi
 
 
-if [[ "${COPY_INTELLIJ_ENV_VARS}" == "y" ]]; then
-  VARIABLE_NAMES=(AZURE_SYNAPSE_WORKSPACENAME AZURE_CREDENTIALS_HOMETENANTID AZURE_CREDENTIALS_APPLICATIONID AZURE_CREDENTIALS_SECRET AZURE_SYNAPSE_SQLADMINUSER AZURE_SYNAPSE_SQLADMINPASSWORD AZURE_SYNAPSE_ENCRYPTIONKEY GOOGLE_APPLICATION_CREDENTIALS GOOGLE_SA_CERT RBS_POOLID RBS_INSTANCEURL)
+VARIABLE_NAMES=(AZURE_SYNAPSE_WORKSPACENAME AZURE_CREDENTIALS_HOMETENANTID AZURE_CREDENTIALS_APPLICATIONID AZURE_CREDENTIALS_SECRET AZURE_SYNAPSE_SQLADMINUSER AZURE_SYNAPSE_SQLADMINPASSWORD AZURE_SYNAPSE_ENCRYPTIONKEY GOOGLE_APPLICATION_CREDENTIALS GOOGLE_SA_CERT RBS_POOLID RBS_INSTANCEURL)
 
-  # Initialize an empty string
+if [[ "${COPY_INTELLIJ_ENV_VARS}" == "y" ]]; then
   SETTINGS=""
-
-  # Loop over the array
-  for VAR_NAME in "${VARIABLE_NAMES[@]}"
-  do
+  for VAR_NAME in "${VARIABLE_NAMES[@]}"; do
     # Append the variable name and its value to the string
     SETTINGS+="$VAR_NAME=${!VAR_NAME};"
   done
 
   # Copy variables to clipboard
+  echo "# Environment variables copied to clipboard"
   echo "$SETTINGS"  | pbcopy
-  echo "Environment variables copied to clipboard"
+fi
+
+if [[ "${ECHO_SETTINGS_TO_OUTPUT}" == "y" ]]; then
```

Comment:
Quotes aren't needed around `y` here. Also if we use `true` and `false`, then you can simplify the `if`
```suggestion
ECHO_SETTINGS_TO_OUTPUT=false

...
   ECHO_SETTINGS_TO_OUTPUT=true
...

if "${ECHO_SETTINGS_TO_OUTPUT}"; then
```